libdnf5/base/log_event.cpp

```diff
@@ -287,6 +330,13 @@ std::string LogEvent::to_string(
                 _("Error: It is not possible to switch enabled streams of a module unless explicitly enabled via "
                   "configuration option module_stream_switch."));
         }
+        case GoalProblem::EXTRA: {
+            return ret.append(utils::sformat(
+                _("Extra package '{}' (with action '{}') which is not present in the stored transaction was pulled "
```

Comment:
May I ask you to mark an order of arguments `{1}`.

Reply:
Added.